python/lsst/daf/butler/datastores/file_datastore/get.py

```diff
@@ -154,6 +175,8 @@ def _read_artifact_into_memory(
         The registry information associated with this artifact.
     isComponent : `bool`
         Flag to indicate if a component is being read from this artifact.
+    cache_manager: `AbstractDatastoreCacheManager`
+        The cache manager to use for caching retrieved files
```

Comment:
The lack of coverage predates this change, see https://app.codecov.io/gh/lsst/daf_butler/blob/main/python%2Flsst%2Fdaf%2Fbutler%2Fdatastores%2FfileDatastore.py#L1459

And the client isn't using caching yet which is why coverage wasn't added here.

Reply:
As discussed yesterday, I think it's because the MetricsExample is not on the allowed caching list. It's likely that we changed a test at some point which changed the storage class. I can take a look at that.